com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ApplicationsCloudModelBuilder.java

```diff
@@ -249,6 +253,36 @@ protected ServiceKeyToInject getServiceKeyToInject(RequiredDependency dependency
         return xsPlaceholderResolver.resolve(applicationDomains);
     }
 
+    protected Map<String, Boolean> getRestartParameters(Module module) {
+        Map<String, Boolean> restartParameters = new HashMap<>();
+        Map<String, Boolean> restartParametersFromDescriptor = getRestartParametersFromDescriptor(module);
+        if (restartParametersFromDescriptor == null) {
+            setRestartParameters(restartParameters, true, true, true);

```

Comment:
The default values are duplicated on lines 263, 264 and 266. Put them in the RestartParameters POJO to avoid this.